### PR TITLE
feat: home page redesign

### PR DESCRIPTION
**Pull request by 0xpatrickdev**
refs: #998

## Summary 

- use default vitepress theme (`layout: home`), styled for agoric

## To Do

- [ ] copy/content and links
- [ ] unfurl
    - We might want to include `og:image` and `twitter:card` meta tags out as part of this PR so we get a nice unfurl: https://github.com/Agoric/documentation/issues/997
- [ ] footer links

## Screenshot

image

### CODE REVIEW

**Comment by dckc** [comment]
for PRs like this with visual impact, screenshots are more than welcome!

image

File: main/.vitepress/styles/index.styl [hunk 1]
```diff
@@ -157,36 +157,6 @@ div[class*="language-"].secondary
...
-
-  @media (max-width: 768px)
-    h1
-      font-size: 1.75rem
-    h3
-      font-size: 1.1rem
-    a
-      color: rgb(171, 35, 40)
-      font-weight: 500
-
```
Comment (0xpatrickdev) [inline main/.vitepress/styles/index.styl:189]: This was a light pass - seems there's more to remove like `.home-section`, `.youtube-embed`

**Comment by 0xpatrickdev** [comment]
We might want to knock this out as part of this: https://github.com/Agoric/documentation/issues/997

File: main/.vitepress/config.mjs [hunk 2]
```diff
@@ -510,5 +513,30 @@ export default defineConfig({
         },
       },
     },
+    footer: {
+      copyright: `© ${new Date().getFullYear()} Agoric Systems Operating Company. All Rights Reserved.`,
```
Comment (dckc) [inline main/.vitepress/config.mjs:517]: is this copyright year computed at build time? or does it rely on the client to compute it?

File: main/.vitepress/config.mjs [hunk 3]
```diff
@@ -510,5 +513,30 @@ export default defineConfig({
...
+    },
+    socialLinks: [
+      {
+        icon: 'discord',
+        ariaLabel: 'Discord',
+        link: 'https://discord.com/invite/qDW8DRes4s',
```
Comment (dckc) [inline main/.vitepress/config.mjs:523]: those invitations expire. IIRC, we have a URL in our own domain for this:

```suggestion
        link: 'https://agoric.com/discord',
```

File: main/.vitepress/config.mjs (hunk 2 above)
Comment (0xpatrickdev) [reply main/.vitepress/config.mjs:517]: I believe this happens at build time

**Review by dckc — APPROVED** [review]
Let's go with this; it can perhaps be improved, but it's clearly better than the status quo.

**Comment by aj-agoric** [comment]
Marking as "done" in the project board since the PR is merged

**Pull request merged**